Expensify/App
# [User Settings] Add new Preferences view

### Details
- Add new toggle on the settings/preferences page to allow users to toggle their `expensifyNewsStatus` value

### Fixed Issues
https://github.com/Expensify/Expensify.cash/issues/1677

### Tests
1. Go to profile by clicking your avatar
2. Click on preferences
3. Toggle and confirm `expensifyNewsStatus` is being updated

### Tested On


### Video

https://user-images.githubusercontent.com/12075600/110719024-f9d32380-81c0-11eb-9282-314b23ea4394.mov

#### Web

![Screen Shot 2021-03-09 at 3 50 48 PM](https://user-images.githubusercontent.com/12075600/110719045-05264f00-81c1-11eb-83b1-a7728aa1e9ab.png)

#### Mobile Web

<img width="494" alt="Screen Shot 2021-03-09 at 3 51 38 PM" src="https://user-images.githubusercontent.com/12075600/110719068-0c4d5d00-81c1-11eb-94b5-6ab8063fc39e.png">

#### Desktop

<img width="1312" alt="Screen Shot 2021-03-09 at 3 52 50 PM" src="https://user-images.githubusercontent.com/12075600/110719097-1707f200-81c1-11eb-9ef3-846f7d6f2ce3.png">

#### iOS

<img width="494" alt="Screen Shot 2021-03-09 at 3 48 42 PM" src="https://user-images.githubusercontent.com/12075600/110719126-20915a00-81c1-11eb-9401-59d061b296d0.png">

#### Android

<img width="464" alt="Screen Shot 2021-03-09 at 3 51 54 PM" src="https://user-images.githubusercontent.com/12075600/110719145-27b86800-81c1-11eb-8759-99f4407e412c.png">
